ui/src/criteria/attribute.tsx

```diff
@@ -46,29 +71,64 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   generateFilter() {
-    if (this.data.selected.length === 0) {
-      return null;
-    }
+    let operands: tanagra.Filter[] | undefined;
+    if (isValid(this.data.min) && isValid(this.data.max)) {
+      operands = [
+        {
+          binaryFilter: {
+            attributeVariable: {
+              variable: "person",
+              name: this.data.attribute,
+            },
+            operator: tanagra.BinaryFilterOperator.LessThan,
+            attributeValue: {
+              int64Val: this.data.max,
+            },
+          },
+        },
+        {
+          binaryFilter: {
+            attributeVariable: {
+              variable: "person",
+              name: this.data.attribute,
+            },
+            operator: tanagra.BinaryFilterOperator.GreaterThan,
+            attributeValue: {
+              int64Val: this.data.min,
+            },
+          },
+        },
+      ];
 
-    const operands = this.data.selected.map(({ id }) => ({
-      binaryFilter: {
-        attributeVariable: {
-          variable: "person",
-          name: this.data.attribute,
+      return {
+        arrayFilter: {
+          operands: operands,
+          operator: tanagra.ArrayFilterOperator.And,
         },
-        operator: tanagra.BinaryFilterOperator.Equals,
-        attributeValue: {
-          int64Val: id,
+      };
+    } else if (this.data.selected.length >= 0) {
```

Comment:
This code indicates that `.selected` and `.min` +  `.max` are mutually exclusive. This should be made clear in the type documentation at L26.